whisper.cpp

```diff
@@ -3387,7 +3387,7 @@ static int timestamp_to_sample(int64_t t, int n_samples) {
     return std::max(0, std::min((int) n_samples - 1, (int) ((t*WHISPER_SAMPLE_RATE)/100)));
 }
 
-static int64_t sample_to_timestamp(int i_sample) {
+static int64_t sample_to_timestamp(int64_t i_sample) {
     return (100*i_sample)/WHISPER_SAMPLE_RATE;
```

Comment:
Great find!
I think this actually solves #188 

Wouldn't it be better to make the cast via the `100ll` literal like this:

```suggestion
static int64_t sample_to_timestamp(int i_sample) {
    return (100ll*i_sample)/WHISPER_SAMPLE_RATE;
```

Reply:
That's actually how I implemented it at first, but I didn't see many other uses of literal suffixes in the codebase, so I changed it.

Updated the PR.